### PR TITLE
Power support for  jicofo

### PR DESCRIPTION
**Pull request by asellappen**
added new yml for the package to support ppc64le and travis build

### CODE REVIEW

**Comment by asellappen** [comment]
know issue :- https://github.com/jitsi/jicofo/issues/581 for mvn failure

**Comment by gururajrkatti** [comment]
Hi @asellappen you can add a comment in travis file for the reason for installing mvn manually. Is it required for intel also ?  
@seth-priya there is failure in the build and an open issue exist.  @asellappen has used ignore failure option for the build. Is this okay to go head ?

**Comment by seth-priya** [comment]
yes, for now its ok, this dependency https://github.com/jitsi/jitsi-sctp#rebuilding-a-new-jni-lib needs to be built and installed from source to resolve the failure and we have been able to validate it but don't have an updated build script published for that yet and I am not sure how easy it would be to integrate those steps into your travis file.

**Comment by asellappen** [comment]
@gururajrkatti, Added comment in  travis file,pl go-ahead .

File: travis-ymls/jicofo.travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+dist : focal
+
+arch:
+  - amd64
+  - ppc64le
+
+before_install: 
+  - mkdir -p /opt/maven
+  - curl https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz|tar -xz --strip 1 -C /opt/maven
```
Comment (gururajrkatti) [inline travis-ymls/jicofo.travis.yml:28]: What is the reason for installing mvn explicitly ? It should be automatically there in travis right ?

File: travis-ymls/jicofo.travis.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - export MAVEN_HOME=/opt/maven
+  - export PATH=${MAVEN_HOME}/bin:${PATH}
+
+script:
+  - mvn test  -Dmaven.test.failure.ignore=true # known issue :- jitsi/jicofo#581 for mvn failure on ppc64le only and -D option can be removed once this issue is fixed
```
Comment (gururajrkatti) [inline travis-ymls/jicofo.travis.yml:33]: Use the ignore option only for ppc64le and not for amd64

**Comment by asellappen** [comment]
@gururajrkatti , done as per revew,Pl check now

**Review by gururajrkatti — APPROVED** [review]
LGTM

**Pull request merged**